source/retryable-reads/tests/estimatedDocumentCount-serverErrors.yml

```diff
@@ -1,9 +1,11 @@
 runOn:
     -
         minServerVersion: "4.0"
+        maxServerVersion: "4.8.99"
```

Comment:
I propose adding a "-pre4.9" suffix to this filename. Likewise for `estimatedDocumentCount.yml`.

Alternatively, suffix both sets of files with the API used internally (e.g. "-count_cmd", "-aggregate_cmd").

This will ensure that anyone coming across these tests later on immediately knows what they pertain to, and won't mistake `estimatedDocumentCount.yml` (or this one) as the main/current tests for that method.

Note: I realize we don't have prior art in place that would have suggested doing this. The closest thing I can think of is CRUD v2 tests where certain methods started taking a `hint` option and we needed to split them into separate files with "-hint" suffixes. That said, this shouldn't be a concern with the unified format going forward since we can simply use test-level `runOnRequirements` (as you did in the new CRUD tests within this PR).

Reply:
Ah that sounds good! I've added the "-pre4.9" suffix to this file, its JSON, `estimatedDocumentCount.yml`, and its JSON.

And yeah, this was much simpler in the one versioned api test that uses `estimatedDocumentCount`, as I could use test-level `runOnRequirements` since it was in unified format.